### ISSUE TITLE
Missing home page messages

### ISSUE THREAD

**Issue by GregLeonhardt** [ryanhowdy/fcms#637]
I am starting with a bare mettle install of Ubuntu 20.04. After update and upgrade the LAMP services were installed followed by the FCMS web site.  Next I start adding messages to the home page.  Everything works as expected up through the 30th message.  However the 31st message and anything following it is never displayed.  I can still create 'Family News' and 'Message Board' which are both displayed.
I also looked in the database and the missing home page messages are there, they are just not being displayed.

OS - Ubuntu 20.04
PHP - 7.4.3
Apache 2.4.41
MariaDB - 15.1

The only thing in the log is:

2022-12-17 11:52:09 - ERROR --> Function get_magic_quotes_gpc() is deprecated
  FILE  - /var/www/html/fcms-3.8.0/familyconnections/fcms.php [187]
  PHP   - 7.4.3 (Linux)
  STACK
    #0 require called at [/var/www/html/fcms-3.8.0/familyconnections/home.php:22]
    #1 fixMagicQuotes called at [/var/www/html/fcms-3.8.0/familyconnections/fcms.php:28]
    #2 fcmsErrorHandler called at [/var/www/html/fcms-3.8.0/familyconnections/fcms.php:187]

This error is also being shown on the front page.  

I would be happy to run tests for you on this or any other issues.

**Comment by ryanhowdy** [comment]
This is a php 5 issue and will be fixed in 4.0.0

**Comment by GregLeonhardt** [comment]
Any idea when the fix will be released?

**Comment by ryanhowdy** [comment]
Not really, but you can check out the progress [here](https://github.com/users/ryanhowdy/projects/3).
